src/Domain/UseCase/GetSettings/GetSettingsResponse.ts

```diff
@@ -0,0 +1,7 @@
+import { Setting } from '../../Setting/Setting'
+import { Uuid } from '../../Uuid/Uuid'
+
+export type GetSettingsResponse = {
+  userUuid: Uuid,
+  settings: Setting[],
```

Comment:
perhaps we should return a slightly different type than `Setting`
in particular, the `user` field should not be a promise (as it won't be serialized to JSON properly) -- it should either be omitted or replaced with user uuid

Reply:
I think `user` field can be omitted from the serialization at this stage - we know that the only person capable of downloading the setting is the owner user.

the reason for it being a promise: https://typeorm.io/#/eager-and-lazy-relations/lazy-relations

One way to overcome this is to introduce projectors, example: https://github.com/standardnotes/auth/blob/9f885ebf7603d687bfd6577a3af90bdd1e8b223c/src/Domain/Auth/AuthResponseFactory20200115.ts#L47